test/test_run/test_integration.py

```diff
@@ -48,11 +48,20 @@ def cleanup_output(cleanup_enabled):
             FILE_MANAGER.get_path("rmd-autograder/submission/tests"),
             FILE_MANAGER.get_path("rmd-autograder/submission/__init__.py"),
             FILE_MANAGER.get_path("rmd-autograder/submission/.OTTER_LOG"),
+            FILE_MANAGER.get_path("qmd-autograder/results/results.json"),
+            FILE_MANAGER.get_path("qmd-autograder/results/results.pkl"),
+            FILE_MANAGER.get_path("qmd-autograder/__init__.py"),
+            FILE_MANAGER.get_path("qmd-autograder/submission/test"),
+            FILE_MANAGER.get_path("qmd-autograder/submission/tests"),
+            FILE_MANAGER.get_path("qmd-autograder/submission/__init__.py"),
+            FILE_MANAGER.get_path("qmd-autograder/submission/.OTTER_LOG"),
         ])
         with FILE_MANAGER.open("autograder/source/otter_config.json", "w") as f:
             f.write(cpy)
         with FILE_MANAGER.open("rmd-autograder/source/otter_config.json", "w") as f:
             f.write(crmd)
+        with FILE_MANAGER.open("qmd-autograder/source/otter_config.json", "w") as f:
+            f.write(cqmd)
```

Comment:
make sure to read the contents of this file and create the `cqmd` variable before the `yield` statement above.